packages/host/app/components/card-renderer.gts

```diff
@@ -17,24 +15,20 @@ import {
   type getCardCollection,
 } from '@cardstack/runtime-common';
 
-import type StoreService from '@cardstack/host/services/store';
-
 import type {
   BaseDef,
   Format,
   Field,
+  CardContext,
 } from 'https://cardstack.com/base/card-api';
 
-import PrerenderedCardSearch from './prerendered-card-search';
-
 interface Signature {
   Element: any;
   Args: {
     card: BaseDef;
     format?: Format;
     field?: Field;
     codeRef?: ResolvedCodeRef;
-    cardContext?: Record<string, any>;
```

Comment:
im not convinced that there is a direct use for passing in this argument when we are already using provider pattern

Reply:
i agree